build/bin/vars

```diff
@@ -2,9 +2,15 @@ bin_dir=$(cd $(dirname $0); pwd -P)
 root_dir=$(cd $(dirname $0)/../..; pwd -P)
 tmp_dir=${root_dir}/tmp
 branch=$(git rev-parse --abbrev-ref HEAD)
-real_pmm_version=$(cat ${root_dir}/VERSION)
+
+# In VERSION file we can have numeric value like '2.0.0' as well as
+# alphanumeric value like '2.0.0-alpha3' which we can not be used entirely
+# e.g. in Version directive in spec files. That is why we split it
+# in two values.
+real_pmm_version=$(cat ${root_dir}/VERSION | awk -F'-' '{print $1}')
+real_pmm_release=$(cat ${root_dir}/VERSION | awk -F'-' '{print $2}')
 if [ -z "${pmm_version}" ]; then
-    if [[ ${branch} == "master" ]]; then
+    if [[ ${branch} == 'PMM-2.0' ]]; then
```

Comment:
That condition was always false, and we were fine with it. Now it will be true for PMM-2.0 branch. Is it what we desire?

Reply:
For master branch we have the same approach. Why do we need to change it for 'PMM-2.0'?

Reply:
Because we want PMM 2.0 to better than PMM 1.x. Including build scripts.

Reply:
@AlekSi want better - requires refactoring. And refactoring should be done separately from other tasks

Reply:
See https://github.com/Percona-Lab/pmm-submodules/pull/267